src/app/components/forms/community/ban/block.vue

```diff
@@ -20,6 +22,12 @@
 				/>
 			</app-form-control-errors>
 		</app-form-group>
+		<template v-else>
+			<label class="control-label"><translate>Username</translate></label>
+			<div class="form-control -locked-username">
+				{{ formModel.username }}
+			</div>
+		</template>
```

Comment:
we should be able to just add the disabled attribute to the app-form-control yeah?
is there a specific reason you go through the trouble of not making this an app form control?

also, i know this can't happen currently, but not having an app-form-group and app-form-control-errors implies the username we initialized the modal with has to be correct since theres no longer a way to surface errors with this field through the form.

Reply:
I didn't do it because I looked for a `disabled` prop on the form control and couldn't find one.
That's because there isn't one.

I didn't want to go through the trouble of adding it to form-control, cause you know, beeg control.

But I did it with a new commit, so should be allgud now.